src/applications/static-pages/addIdToSubHeadings.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Add unique ID to H2s and H3s that aren't in WYSIWYG or accordion buttons
+ */
+
+export function uniqueId() {
+  return `id-${Math.random()
```

Comment:
I'm not sure I understand the use of these ids if they're completely random. No one will be able to link to them because they'll be changing every build.

What about using the algorithm that GH uses for headings? https://gist.github.com/asabaylus/3071099#gistcomment-1593627.

Reply:
Used that GH algorithm to make unique, semantic ids